### ISSUE TITLE
Reflection Padding?

### ISSUE THREAD

**Issue by im9uri** [lengstrom/fast-style-transfer#29]
In Johnson's [paper](https://arxiv.org/abs/1603.08155), the input is padded with via reflection padding.

In this repo, reflection padding is not implemented, and padding is also used in the residual blocks.

Is there a reason the implementation is different from Johnson's paper? Or am I looking at the code in the wrong way?

Thanks.
Gyuri.

**Comment by Keloo** [comment]
Check this out: https://github.com/lengstrom/fast-style-transfer/pull/56

**Comment by XHUJOY** [comment]
Change `net = tf.nn.conv2d(net, weights_init, strides_shape, padding='SAME')` to
```
padding = [[0, 0], [filter_size // 2, filter_size // 2],
           [filter_size // 2, filter_size // 2], [0, 0]]
net = tf.pad(net, padding, 'REFLECT')
net = tf.nn.conv2d(net, weights_init, strides_shape, padding='VALID')
```
and do the same thing for  tf.nn.conv2d_transpose in transform.py

**Comment by im9uri** [comment]
Thanks everyone!
